src/app.rs

```diff
@@ -154,7 +154,9 @@ pub fn run(
                         selected_gpu = usize::from(n - 1)
                     }
                     KeyCode::Char('p') => {
-                        // re-scan pci tree to let driver discover new devices (only works as sudo)
+                        //commented out because discover_gpus not implemented
```

Comment:
It is implemented https://docs.rs/nvml-wrapper/latest/nvml_wrapper/struct.Nvml.html#method.discover_gpus, it's probably not available on the platform (laptop gpus).

We need a better way to conditionally show/not-show this for sure.

Reply:
i was not able to compile, i will look again

Reply:
```rust
/* snip

# Platform Support

    Only supports Linux.
    */
    // TODO: constructor for default pci_infos ^
    // Checked against local
    // Tested
    #[cfg(target_os = "linux")]
    #[doc(alias = "nvmlDeviceDiscoverGpus")]
    pub fn discover_gpus(&self, pci_info: PciInfo) -> Result<(), NvmlError> {
        let sym = nvml_sym(self.lib.nvmlDeviceDiscoverGpus.as_ref())?;

        unsafe { nvml_try(sym(&mut pci_info.try_into()?)) }
    }

/* snip */
```

as it writes, this function is only for linux, im testing it on windows. so i will try to not show "p to rescan" on windows

Reply:
Good call, I too should've gone deeper in the docs before snarking, soz.